DTS-STN/Alpha-Site
# Feature/#127 alpha banner

# Description

[MVP Alpha header banner] https://trello.com/b/ZqWtJSyh/alpha-site-board

This is a visual feature. It is the Alpha banner at the top of every page.
![image](https://user-images.githubusercontent.com/77116011/114773093-98aceb80-9d3c-11eb-8f44-1b572ddededb.png)

## Acceptance Criteria

N/A

## Test Instructions

1. Go to branch name feature/#127-AlphaBanner
2. npm run dev
3. See the banner at the top of the page.
4. Test multiple screen size width.

## Help Requested

Feedback is always appreciated.

## Checklist

- [ x] Strings use placeholders for translation (No hard coded strings)
- [ x] Unit tests have been added/updated

## Product and Sprint Backlog

[MVP Alpha header banner] https://trello.com/b/ZqWtJSyh/alpha-site-board
